src/osg/Texture2DMultisampleArray.cpp

```diff
@@ -0,0 +1,150 @@
+/* -*-c++-*- OpenSceneGraph - Copyright (C) 1998-2010 Robert Osfield
```

Comment:
We may want to incorporate the new code for 3.6 that is in Texture2DMultisample.cpp into this file as well.